src/main/java/org/jenkins/tools/test/model/hook/PluginCompatTesterHooks.java

```diff
@@ -76,9 +75,11 @@ private void setupExternalClassLoaders(List<File> externalJars) throws Malformed
         if (externalJars == null) {
             return;
         }
+        List<URL> urls = new ArrayList<>();
```

Comment:
This works.  or we could just make any callers add the external hook jars to the classpath when starting the pct and then remove some extra complexity.

This looks like a good enough improvement though.